common/beerocks/bwl/dwpal/mon_wlan_hal_dwpal.cpp

```diff
@@ -39,6 +44,38 @@ namespace dwpal {
             delete[] obj;                                                                          \
     })
 
+// Helper macro for dwpal_nl_cmd_get
+// interfaceName, out_buffer, out_buffer_size, max_buffer_size, paramsObj
+#define DWPAL_GET_SCAN_PARAMS_HELPER_FG(IFNAME, OUT_BUFF, OUT_BUFF_SZ, MAX_BUFF_SZ, PRMS)          \
```

Comment:
I really don't like these style of MACROs especially that we have c++11. Please create a static inline functions and use them instead.

Reply:
These macros just obfuscate the control flow. No static inline functions, just keep the control flow explicit in the calling function.